tests/unittests/test_mock_blob_shared_memory_functions.py

```diff
@@ -54,6 +54,7 @@ async def test_binary_blob_write_function(self):
         func_name = 'put_blob_as_bytes_return_http_response'
         async with testutils.start_mockhost(script_root=self.blob_funcs_dir) \
                 as host:
+            await host.init_worker("4.17.1")
```

Comment:
Where is the `init_worker` method defined?

Reply:
Its in testutils. It calls the worker_init_request. Without this call the host.load_function fails since the binding registry does not get initialized